Добавлен класс сущность Meals.

lasm16 opened the pull request:
Добавлены конфигурации модели для класса Meal.
Еще не закончил, есть что добавить.

herobeniyoutube committed:
Регистрация ef core в di
Временный хотсед сервис для проверки функциональности бд
Докинул на nutrition атрибуты
Контекст отредачил для использования options + перевод CamelCase нашего в snake_case постгре в OnModelCreation

herobeniyoutube committed:
Merge branch 'dev' into feature/OTUS-3

Vitaly0728 committed:
Добавил  модели для тренера

Commit:
identity BigUp

noxyMane666 committed:
Identity Bigup

noxyMane666 committed:
Delete AiWebGymTracker/.idea/.idea.AiWebGymTracker.dir/.idea directory

Убраны лишнии папки

noxyMane666 committed:
Delete global.json

Убрано лишнее

noxyMane666 committed:
Delete .DS_Store

Убрано лишнее

noxyMane666 committed:
Delete .idea/.idea.AiWebGymTracker/.idea directory

Убрано лишнее

Commit:
Добавлен класс сущность Meals.
Добавлены конфигурации модели для класса Meal.
Еще не закончил, есть что добавить.

Commit:
Добавлен класс сущность Meals.
Добавлены конфигурации модели для класса Meal.
Еще не закончил, есть что добавить.

noxyMane666 committed:
Fixes

noxyMane666 committed:
Merge remote-tracking branch 'origin/nagimov_dev' into nagimov_dev

noxyMane666 committed:
Delete AiWebGymTracker/appsettings.Development.json

appSettings

noxyMane666 committed:
Delete AiWebGymTracker/appsettings.json

appSettings

Xam1l1on commented on AiWebGymTracker/Models/Meals.cs:
```diff
@@ -0,0 +1,31 @@
+    public DateTime UpdatedAt { get; set; } = DateTime.Now;
+    public string CreatedBy { get; set; }
+    public string UpdatedBy { get; set; }
+    public MealCategory MealCategory { get; set; }
```
Убрал класс

Xam1l1on commented on AiWebGymTracker/Models/Meals.cs:
```diff
@@ -0,0 +1,31 @@
+    public int MealCategoryId { get; set; }
+    public int UserId { get; set; }
+    public User User { get; set; }
+    public MealCategory MealCategory { get; set; }
```
Убрал 
```suggestion
    public MealCategory MealCategory { get; set; }
```

Xam1l1on committed:
Изменил название класса Meal.cs, теперь Food.cs
Убрал ModelBuilder из ApplicationContext.cs
Удалил класс MealType.cs
Создал enum FoodCategory в классе Food.cs - содержит категорию продуктов

Xam1l1on committed:
Изменил название класса Meal.cs, теперь Food.cs
Убрал ModelBuilder из ApplicationContext.cs
Удалил класс MealType.cs
Создал enum FoodCategory в классе Food.cs - содержит категорию продуктов

Vitaly0728 committed:
Изменениия  после рефакторинга

lasm16 committed:
hotfix

Vitaly0728 committed:
правка замечаний

noxyMane666 committed:
Review Updates

noxyMane666 committed:
Merge remote-tracking branch 'origin/nagimov_dev' into nagimov_dev

# Conflicts:
#	AiWebGymTracker/appsettings.json

noxyMane666 committed:
Code Review

lasm16 committed:
Merge pull request #7 from lasm16/nagimov_dev

Nagimov dev

lasm16 committed:
Merge branch 'dev' into feature/OTUS-3

lasm16 committed:
Merge pull request #3 from lasm16/feature/OTUS-3

Feature/otus 3

herobeniyoutube committed:
otus 22 таблица nutrition sets

lasm16 committed:
hotfix. Add correct db context

lasm16 committed:
Merge pull request #4 from lasm16/feature/OTUS-22

nutrition sets добавлена

lasm16 committed:
Merge pull request #5 from lasm16/feature/OTUS-12

Добавил  модели для тренера

lasm16 committed:
Merge pull request #9 from lasm16/master

Косяк Виталия

lasm16 committed:
hotfix. Review fixes

lasm16 committed:
review fix

lasm16 committed:
fix
